WalletWasabi.Fluent/Views/Wallets/Receive/ReceiveAddressView.axaml

```diff
@@ -39,7 +39,7 @@
         </c:PrivacyContentControl>
       </StackPanel>
 
-      <c:AdorningContentControl MaxHeight="270" HorizontalAlignment="Center" ClipToBounds="False">
+      <c:AdorningContentControl HorizontalAlignment="Center" VerticalAlignment="Center" ClipToBounds="False">
         <c:AdorningContentControl.Adornment>
           <Panel Margin="0 0 -5 20">
             <Button Command="{Binding SaveQrCodeCommand}" ToolTip.Tip="Save" Classes="plain icon"
```

Comment:
We can collapse this tag for readability

Reply:
I don't understand what you mean. Can you elaborate?

Reply:
Sorry, my mistake. Please ignore.